tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "ES2020",
```

Comment:
This time I made vuepress support `optional chaining operator` by lowering the target version.

If it is not proper, please tell me and I will find another way to make it.

Reply:
Thanks for looking into what is causing the vuepress build not to work. I didn't know that setting affects vuepress.

Reply:
Sorry, I just lost my mind at that moment 😢. This doesn't affect `js` files, errors just disappear accidentally.